packages/connect/src/internals.js

```diff
@@ -1,3 +1,23 @@
 export const proxyToRaw = new WeakMap();
-export const rawToProxy = new WeakMap();
+// export const rawToProxy = new WeakMap();
 export const rawToRoot = new WeakMap();
```

Comment:
`rawToRoot` should be removed. We can store a global reference to the root somewhere.